packages/grid/x-grid/README.md

```diff
@@ -15,11 +15,12 @@ npm install @mui/x-data-grid-pro
 yarn add @mui/x-data-grid-pro
 ```
 
-This component has two peer dependencies that you will need to install as well.
+This component has three peer dependencies that you will need to install as well.
```

Comment:
This makes this PR breaking no ?

Reply:
No, the styles package is already mentioned as a peer dependency in https://material-ui.com/components/data-grid/getting-started/#installation. This change is only syncing the README with the docs.